docs/asset_info.md

```diff
@@ -184,82 +196,125 @@ Response:
- “Links”:  [
-
-“Activities”:  ”https://activitiesapi.hackney.gov.uk/propertyactivities/[propertyId]”,
-
- “Documents”:  ”https://documentsapi.hackney.gov.uk?type=property&id=[propertyId]”,
-
- “EPC Certificate” : “https://assetapi.hackney.gov.uk/epc-cert/[00001111]”
-
- “Tenancies”: “https://tenanciesapi.hackney.gov.uk/tenancies[propertyId]”,
-
- “Facilities”: “https://assetsapi.hackney.gov.uk/facilities/[00001111]”
-]
-}
+       {
+              "Id": "6f22e9ae-3e8a-4e0e-af46-db02eb87f8e6",
+              "assetId": "00007918",
+              "assetType": "Garage | Dwelling | Estate ...",
+              "assetLocation": {
+                     "parentAssets": [
+                     {
+                            "id": "6f22e9ae-3e8a-4e0e-af46-db02eb87f8e6",
+                            "type": "sub-block",
+                            "name": "Newcombe House (1-18)"
+                     },
+                     {
+                            "id": "6f22e9ae-3e8a-4e0e-af46-db02eb87f8e6",
+                            "type": "block",
+                            "name": "Newcombe House (1-35)"
+                     },
+                     {
+                            "id": "6f22e9ae-3e8a-4e0e-af46-db02eb87f8e6",
+                            "type": "estate",
+                            "name": "Newcombe House Estate Powell Road"
+                     }
+                     ],
+                     "floor": "G",
+                     "totalBlockFloors": "5"
+              },
+              "assetAddress": {
+                     "uprn": "100021065786",
+                     "addressLine1": "Powell Road",
+                     "addressLine2": "Hackney",
+                     "addressLine3": "London",
+                     "addressLine4": "",
+                     "postCode": "E5 8DH",
+                     "postPreamble": "1 Newcome House"
+              },
+              "assetManagement": {
+                     "agent": "HAH",
+                     "areaOfficeName": "Clapton Panel Area Team",
+                     "isCouncilProperty": false,
+                     "managingOrganisation": "LBH, -- Hackney or TMO",
+                     "managingOrganisationId": "6f22e9ae-3e8a-4e0e-af46-db02eb87f8e6",
+                     "owner": "London Borough of Hackney",
+                     "isTMOManaged": true,
+                     "propertyOccupiedStatus" : "Vacant",
+                     "isNoRepairsMaintenance" : true,
+                     "fundingSource" : "TA",
+                     "costCentre" : "D0720",
+                     "councilTaxType" : "House in multiple occupancy (HMO)",
+                     "councilTaxLiability" : "Void only",
+                     "lhaArea" : "Inner East London",
+                     "isTemporaryAccomodation" : true,
+                     "readyToLetDate" : true,
+                     "assetStatus" : "House",
```

Comment:
I believe this field is already covered by 'propertyOccupiedStatus' as mentioned in the spreadsheet.

Reply:
Agreed, removed this assetStatus field now